packages/qdrant-loader/src/qdrant_loader/connectors/jira/data_center_connector.py

```
@@ -0,0 +1,139 @@
+
+                    if (start_at + i + 1) % progress_log_interval == 0:
+                        progress_percent = (
+                            round((start_at + i + 1) / total_issues * 100, 1)
+                            if total_issues > 0
+                            else 0
+                        )
+                        logger.info(
+                            f"🎫 Progress: {start_at + i + 1}/{total_issues} issues ({progress_percent}%)"
+                        )
+
+                except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)